docs/resources/bunny_hostname.md

```diff
@@ -0,0 +1,33 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "bunny_hostname Resource - terraform-provider-bunnycdn"
+subcategory: ""
+description: |-
+  
+---
+
+# bunny_hostname (Resource)
+
+
+
+
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- **hostname** (String) The hostname value for the domain name.
+- **pull_zone_id** (Number) The ID of the pull zone to that the hostname belongs.
+
+### Optional
+
+- **id** (String) The ID of this resource.
+
+### Read-Only
+
+- **force_ssl** (Boolean) Determines if the Force SSL feature is enabled.
+- **has_certificate** (Boolean) Determines if the hostname has an SSL certificate configured.
+- **is_system_hostname** (Boolean) Determines if this is a system hostname controlled by bunny.net.
```

Comment:
maybe `is_default_hostname` is better ?

Reply:
That name would definitively more clear.  :-)
I would keep the current name for now because it's the same then in the official official api documentation:
https://docs.bunny.net/reference/pullzonepublic_index.
I also don't know any official reference where it's documented that it is only true for the default pull-zone hostname.

Reply:
I see. Well then ofc stick with the naming provided by the api.